wishlist.css

```diff
@@ -0,0 +1,27 @@
+.card-container{
+
+    min-width: 22rem;
+    height: 27rem;
+}
+
+.img-contain>img{
+
```

Comment:
Remove unwanted space. Otherwise the code looks good to merge

Reply:
sure!